_shared/project/tests/pytest_plugins/factory_boy.py

```diff
@@ -0,0 +1,10 @@
+import factory.random
+import pytest
+
+
+@pytest.fixture(scope="session", autouse=True)
+def factory_boy_random_seed():
```

Comment:
We could use the same mechanism to do the same for the random package:

https://github.com/pytest-dev/pytest/issues/667#issuecomment-112206152